presto-native-execution/src/test/java/com/facebook/presto/nativeworker/TestHiveQueriesGlueMetastore.java

```diff
@@ -0,0 +1,446 @@
+                .put("hive.metastore.glue.default-warehouse-dir", tempWarehouseDir.getCanonicalPath())
+                // This option is required to avoid the error:
+                // "Table scan with filter pushdown disabled is not supported".
+                // Velox requires this flag to be set for scans.
+                .put("hive.pushdown-filter-enabled", "true")
+                // The options below are required to avoid "Access Denied" errors when performing operations such
+                // as DROP TABLE.
+                .put("hive.max-partitions-per-writers", "999")
+                .put("hive.allow-drop-table", "true")
+                .put("hive.allow-rename-table", "true")
+                .put("hive.allow-rename-column", "true")
+                .put("hive.allow-add-column", "true")
+                .put("hive.allow-drop-column", "true")
+                .build();
+
+        queryRunner.createCatalog("test_glue", "glue", glueProperties);
+    }
+
+    @Override
+    protected Session getSession()
+    {
+        return Session.builder(super.getSession())
+                // This option is required to avoid the error:
+                // "Unknown plan node type com.facebook.presto.sql.planner.plan.TableWriterMergeNode".
+                // TableWriterMergeNode is not supported by Velox yet.
+                .setSystemProperty("table_writer_merge_operator_enabled", "false")
+                .build();
+    }
+
+    private static String getRandomName(String prefix)
+    {
+        String randomName = UUID.randomUUID().toString().toLowerCase(ENGLISH).replace("-", "");
+        return TEMPORARY_PREFIX + prefix + randomName;
+    }
+
+    /**
+     * Returns the fully-qualified schema name.
+     */
+    private static String tempSchema(String prefix)
+    {
+        return CATALOG_NAME + "." + getRandomName(prefix);
+    }
+
+    /**
+     * Returns the fully-qualified table name.
+     */
+    private static String tempTable(String schemaName, String prefix)
+    {
+        return schemaName + "." + getRandomName(prefix);
+    }
+
+    @BeforeClass
+    public void setUp() throws IOException
+    {
+        tempWarehouseDir = Files.createTempDir();
+        setUpGlueCatalog(getQueryRunner(), tempWarehouseDir);
+        setUpGlueCatalog((QueryRunner) getExpectedQueryRunner(), tempWarehouseDir);
+
+        schemaName = tempSchema("test_schema");
+        getQueryRunner().execute("CREATE SCHEMA " + schemaName);
+    }
+
+    @AfterClass(alwaysRun = true)
+    public void tearDown() throws IOException
+    {
+        deleteRecursively(tempWarehouseDir.toPath(), ALLOW_INSECURE);
+        if (schemaName != null) {
+            getQueryRunner().execute("DROP SCHEMA IF EXISTS " + schemaName);
+        }
+    }
+
+    @Test
+    public void testShowCatalogs()
+    {
+        // Glue catalog should be configured correctly.
+        MaterializedResult res = getQueryRunner().execute("show catalogs like '%" + CATALOG_NAME + "%'");
```

Comment:
Presto does NOT use abbreviations like `res`. Rename it to `materializedResult` please?

Reply:
Yes, they do: https://github.com/prestodb/presto/blob/master/presto-main/src/main/java/com/facebook/presto/operator/scalar/JsonPath.java#L96, https://github.com/prestodb/presto/blob/master/presto-main/src/test/java/com/facebook/presto/operator/TestSpillableGroupedTopNBuilder.java#L144.

Yes, I will rename.